examples/chat/src/chatScreen.tsx

```diff
@@ -1,21 +1,49 @@
+import { createImage } from "jazz-browser-media-images";
 import { ID } from "jazz-tools";
 import { useState } from "react";
-import { useCoState } from "./main.tsx";
+import { useAccount, useCoState } from "./main.tsx";
 import { Chat, Message } from "./schema.ts";
 import {
   BubbleBody,
   BubbleContainer,
+  BubbleImage,
   BubbleInfo,
+  BubbleText,
   ChatBody,
-  ChatInput,
   EmptyChatMessage,
+  ImageInput,
+  InputBar,
+  TextInput,
 } from "./ui.tsx";
 
 export function ChatScreen(props: { chatID: ID<Chat> }) {
   const chat = useCoState(Chat, props.chatID, [{}]);
+  const { me } = useAccount();
   const [showNLastMessages, setShowNLastMessages] = useState(30);
 
-  return chat ? (
+  if (!chat)
+    return (
+      <div className="flex-1 flex justify-center items-center">Loading...</div>
+    );
+
+  const sendImage = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (!me?.profile) return;
+
+    const file = event.currentTarget.files?.[0];
+
+    if (file) {
+      createImage(file, { owner: me.profile._owner }).then((image) => {
```

Comment:
Not sure why this works (guess that the profile is shared to everyone by default) but would be cleaner to link the image to the chat group:

```suggestion
      createImage(file, { owner: chat._owner }).then((image) => {
```

Reply:
I think I should change the owner of Message to `me.profile._owner` so that, if we add Edit and Delete, then only the sender can perform those.